Projects/Parking Operator/helpers.py

```diff
@@ -0,0 +1,7 @@
+from os import system
+def clear_display():
+
+    try: 
+        system("clear")
```

Comment:
'clear' is not recognized as an internal or external command,
operable program or batch file.